src/vstest.console/Processors/Utilities/ArgumentProcessorFactory.cs

```diff
@@ -237,7 +237,9 @@ public IEnumerable<IArgumentProcessor> GetArgumentProcessorsToAlwaysExecute()
                 new DisableAutoFakesArgumentProcessor(),
                 new ResponseFileArgumentProcessor(),
                 new EnableBlameArgumentProcessor(),
-                new UseVsixExtensionsArgumentProcessor()
+                new UseVsixExtensionsArgumentProcessor(),
+                new ListFullyQualifiedTestsArgumentProcessor(),
+                new ListTestsTargetPathArgumentProcessor()
```

Comment:
Also add acceptance tests for these processors.

Reply:
Done